feat(dropdowns): use svg icon

jefflembeck opened the pull request:
Update that includes using svg icon for dropdown caret +
make sure that the documentation is updated and better looking

![screen shot 2016-06-01 at 12 03 54 pm](https://cloud.githubusercontent.com/assets/109699/15722132/ef33ae38-27f0-11e6-8a40-92b51974dbaf.png)

jefflembeck committed:
feat(dropdowns): use svg icon

Update that includes using svg icon for dropdown caret +
make sure that the documentation is updated and better looking

jefflembeck commented:
cc @stubbornella @jerrygabra how do we feel about this spacing?

stubbornella commented on src/pivotal-ui/components/dropdowns/dropdowns.scss:
```diff
@@ -381,6 +397,18 @@ This is a notification dropdown with alerts.
+  .caret {
+    display: inline-block;
+    border: 0;
+    margin-right: -0.6em;
```
is this always going to match the font correctly? Did it look right with a small button (that has smaller font)?

jefflembeck replied:
Small change on it, but yes!

stubbornella commented on src/pivotal-ui/components/dropdowns/dropdowns.scss:
```diff
@@ -381,6 +397,18 @@ This is a notification dropdown with alerts.
+    margin-left: 5px;
+    margin-top: 0;
+    margin-bottom: 0;
+    font-size: 20px;
```
is it necessary to set the font-size now that we are using svg?

jefflembeck replied:
`width` and `height` are proportionate based upon the `icon` class. In the case of the `caret`, here. I'm inflating a bit. I've changed to `1.25em` in the most recent push. Makes it way better for proportional changes.

stubbornella commented:
looks good, I left a couple small comments

jefflembeck committed:
fix(dropdowns): make spacing better proportionally

This allows big, small, and default sized buttons to live together in peace and
harmony

Also, yay more documentation

woof

jefflembeck commented:
![pivotal ui css dropdowns 2016-06-01 12-27-50](https://cloud.githubusercontent.com/assets/109699/15722807/4ee1374e-27f4-11e6-8e31-8eefd1c5acb3.png)
![pivotal ui css dropdowns 2016-06-01 12-27-59](https://cloud.githubusercontent.com/assets/109699/15722806/4ee0787c-27f4-11e6-9444-441e5fac1c2f.png)

jefflembeck commented:
added more docs (for small button) so we can catch regressions in that in the future

jefflembeck committed:
chore(pui): update dropdowns all together

jefflembeck committed:
fix(dropdowns): update docs with text class

jefflembeck commented:
got the 👍 from @jerrygabra
